modules/permission-sets/variables.tf

```diff
@@ -11,6 +11,10 @@ variable "permission_sets" {
       name = string
       path = string
     }))
+    boundary_policy_attachments = list(object({
```

Comment:
@aaronrea can you make this optional with a default of `[]`. This feels like a solid feature add, but I don't believe 90% of module consumers will have to use this functionality because permission boundaries are only necessary in large environments. Considering that, I want to make the barrier to upgrading + future usage as easy as possible. Keeping it optional with a default feels like it will be available for people, but we won't force them into it. And since we're already on TF 1.3+ here, we can use those features without a version bump requirement. 

Thanks!